webapp/components/PasswordReset/Request.vue

```diff
@@ -124,17 +102,9 @@ export default {
   },
 }
 </script>
-<style>
+<style scoped>
 .login-image {
   width: 90%;
   max-width: 200px;
 }
```

Comment:
if this is not used anymore, we should remove it, although I would like to know the motive for removing the image from these forms... they seems a bit bland

Reply:
I did not really like the logo on the password reset form and would have preferred a "lock" icon or image that distinguishes the form from the signup form etc. And I think the motivation for removing it was that I had troubles to get the layout right.